pkg/logging/logging.go

```diff
@@ -0,0 +1,98 @@
+package logging
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+/*
```

Comment:
Should we use `//` instead of `/**/`? I see the `//` as a more common practice to define comment blocks in Golang.

Reply:
Right... done